adoc/deployment-bootstrap.adoc

```diff
@@ -1,9 +1,58 @@
 [[bootstrap]]
 == Bootstrapping the cluster
 
+Bootstrapping the cluster is the initial process of starting up the cluster
+and defining which of the nodes are masters and which workers. For maximum automation of this process
+{productname} uses the skuba package.
+
+=== Preparation
+
+==== Install skuba
+
+First you need to install skuba on your local workstation:
+
+. Add the SLE15 SP1 extension containing skuba. This also requires the "containers" module.
++
+----
+SUSEConnect -p sle-module-containers/15.1/x86_64
+SUSEConnect -p caasp/4.0/x86_64 -r <Registration Code>
+----
+. Install skuba:
++
+----
+sudo zypper in skuba
+----
+
 === Cluster Deployment
 
-. Initialize the cluster on the deployed machines.
+Make sure you have added the SSH identity (corresponding to the SSH key distributed above) to the ssh-agent on your workstation.
+This is a requirement for skuba (https://github.com/SUSE/skuba#prerequisites).
```

Comment:
just a sidenote, the repo is still private, but going to be publicly available soon